generated/nidaqmx/_task_modules/triggering/start_trigger.py

```diff
@@ -1037,6 +1037,20 @@ def cfg_dig_pattern_start_trig(
         self._interpreter.cfg_dig_pattern_start_trig(
             self._handle, trigger_source, trigger_pattern, trigger_when.value)
 
+    def cfg_time_start_trig(self, when, timescale=Timescale.USE_HOST):
+        """
+        New Start Trigger
+
+        Args:
+            when (nidaqmx.constants.DateTime): Specifies when to
```

Comment:
FYI: Please consider enabling hyperlinks to the stdlib docs for datetime by enabling the intersphinx plugin. See https://github.com/ni/measurementlink-python/pull/430 for an example.

This is not required for this PR. I am posting this suggestion on multiple PRs.

Reply:
I think in the interest of small PRs either @WayneDroid or I will address this in a follow up PR for nidaqmx-python. Thanks for the heads up though!